spells/unlock-station.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable linebreak-style */
+function unlockStationSpell(context, payload) {
+  // check the game phase, only when in play phase
+  const gameState = context.getGameState();
+  const gamePhase = gameState.properties.phase.type;
+  // console.log(gameState.properties.phase.type);
+
+  // there is no game
+  if (gamePhase === null) {
```

Comment:
Change this into checking `gameState === null`, since if you're checking for `gamePhase` and `gameState` happens to be null this wont work. So move line 5 after this if-statement.